### DISCUSSION TITLE
Github education program

### DISCUSSION THREAD

**Discussion by ksawalha** [community/community#147145] [GitHub Education]
### Hi everyone,

Hello everyone! This is my first post on GitHub Education, and I'm looking for help, or at least an explanation of my problem, in order to learn more about how the platform works. I was recently approved to apply for GitHub Education benefits, but I'm still waiting for the availability of the benefits.
![IMG_0140](https://github.com/user-attachments/assets/519cfb48-851f-447e-8ea0-be18b71d2edb)

**Comment by davevad93** [comment]
Hi @ksawalha ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ksawalha was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
